### ISSUE TITLE
Hidden RC logging per default

### ISSUE THREAD

**Issue by xbony2** [FTB-Gamepedia/Tilesheets#54]
I was going to try to work on this to practice some PHP but time is short.

Restricting the creation of tilesheets because you have to use a bot so it won't spam the RC is the dumbest thing ever; making tilesheets should be easy as possible, you shouldn't have to rely on the two people who can (not to mention bots and this don't work right now anyway).

**Comment by elifoster** [comment]
That's not something we have control of at all. Also not sure what you mean by two people since a bunch can do it.

**Comment by xbony2** [comment]
Unless I'm missing something, it's _literally_ two people. If you can convince Retep to create a tilesheet for someone else I'll run for president.

Because the translation extension does something semi-similar, I don't see how it's impossible.

**Comment by elifoster** [comment]
Okay I'm actually awake so I can respond decently now.

>Restricting the creation of tilesheets because you have to use a bot so it won't spam the RC is the dumbest thing ever

This is basically the reason anything is restricted to bot accounts. It's not just a thing our extension does.

>making tilesheets should be easy as possible

Eventually Peter's program is going to import them automatically if I understand his plans correctly.

>you shouldn't have to rely on the two people who can ... Unless I'm missing something, it's literally two people.

I remember fairly distinctly that you, LuminousLizard, Chocohead, myself, Peter, and probably others, also have the ability to create tilesheets.

>not to mention bots and this don't work right now anyway

That is going to be fixed by #55 

>Because the translation extension does something semi-similar, I don't see how it's impossible.

The translation extension uses a sock puppet bot account.

**Comment by xbony2** [comment]
> This is basically the reason anything is restricted to bot accounts. It's not just a thing our extension does.

Yes, but we shouldn't force people to make bot accounts just to do what should be such a basic function of the wiki. It's overcomplicated.

> Eventually Peter's program is going to import them automatically if I understand his plans correctly.

Which is a step in the right direction but not really there (not particularly relevant to this discussion though).

> I remember fairly distinctly that you, LuminousLizard, Chocohead, myself, Peter, and probably others, also have the ability to create tilesheets.

Hold up. You and me, yes. LuminousLizard is daed. Chocohead does not. And like I said if you can convince Retep to create a tilesheet for someone else I'll run for president. As for probably others, feel free to point out anyone.

> That is going to be fixed by #55

yay

> The translation extension uses a sock puppet bot account.

Nope, you've completely missed what I meant. Go to Special:RecentChanges and stare at the page until you see it :P

**Comment by elifoster** [comment]
>Nope, you've completely missed what I meant. Go to Special:RecentChanges and stare at the page until you see it :P

Do you mean the markup log entries? There's like maybe 20 per day on a really active day, so that really isn't an issue. It's not like with tilesheet importing where it can be like 200 entries per sheet.

**Comment by xbony2** [comment]
Yeah you've totally missed it. "Filter translations"? It hides the stuff in the translations namespace?

**Comment by retep998** [comment]
Note that each time someone translates a section of a translatable page, it actually creates two RC entries: one in the Translations namespace (which even my own IRC RC bot filters out), and the other as an edit to the translated page which has no way of being filtered out.
